bricksrc/timeseries.py

```diff
@@ -0,0 +1,57 @@
+"""Defines the timeseries reference model"""
+
+from .namespaces import BRICK, A, OWL, SKOS, SH, XSD
+from rdflib import Literal, BNode
+
+
+def define_timeseries_model(G):
```

Comment:
Why do we need a script to generate this instead of posting a shacl file directly? This script is just a python version of a shacl file, which I don't think is useful to maintain but complicates maintenance. Still, I'm fine if the purpose is to make it consistent with other generation processes.

Reply:
The timeseries model may get more complex in the future, so I made sure to leave us some code that we can extend later